tracer.go

```diff
@@ -82,13 +82,15 @@ func sqlOperationName(query string) string {
 	return strings.ToUpper(parts[0])
 }
 
-func appendConnectionAttributes(config *pgx.ConnConfig, opts []trace.SpanStartOption) {
```

Comment:
oops! good catch! :see_no_evil: 

nit: WDYT of renaming the function, since it's not not really appending to anything, just returning the span-options for the config?

Reply:
@obitech WDYT?

Reply:
Good idea. How about `connectionAttributesFromConfig` ?